core/src/main/java/brooklyn/location/basic/SshMachineLocation.java

```diff
@@ -216,6 +217,13 @@ public String apply(HasConfigKey<?> input) {
                 }
             }));
 
+    /**
+     * The set of config keys on this location which become default values for properties when invoking an SSH
+     * operation.
+     */
+    public static final Set<ConfigKey<?>> SSH_CONFIG_GIVEN_TO_PROPS = ImmutableSet.<ConfigKey<?>>of(
```

Comment:
Good question. I thought it should be `private` but then I saw that similar fields were `public` so I went with that. `@Beta` sounds like a good idea though.